be/src/Unic.UrlMapper2/serialization/Templates/UrlMapper2/Redirect/Redirect Configuration/Preserve Query String.yml

```diff
@@ -0,0 +1,32 @@
+﻿---
+ID: "a9d3d444-c009-4c90-b1a2-f8c25d28d084"
+Parent: "aa0cf857-7540-4fca-830a-7370d9693d23"
+Template: "455a3e98-a627-4b40-8035-e683a0331ac7"
+Path: /sitecore/templates/Modules/UrlMapper2/Redirect/Redirect Configuration/Preserve Query String
+DB: master
+SharedFields:
+- ID: "ab162cc0-dc80-4abf-8871-998ee5d7ba32"
+  Hint: Type
+  Value: Checkbox
+- ID: "ba3f86a2-4a1c-4d78-b63d-91c2779c1b5e"
+  Hint: __Sortorder
+  Value: 550
+- ID: "be351a73-fcb0-4213-93fa-c302d8ab4f51"
+  Hint: Shared
+  Type: Checkbox
+  Value: 1
+Languages:
+- Language: en
+  Fields:
+  - ID: "577f1689-7de4-4ad2-a15f-7fdc1759285f"
+    Hint: __Long description
+    Value: this field also enables regex support
```

Comment:
**thought:** This seems like a very technical description of what is happening and might cause more confusion than help understand the intention/functionality of the field.

**suggestion:** Focus the description more to help the understanding of content authors. Maybe something like:
> Enabling this checkbox will pass along the query string from the requested URL

Reply:
Yep, I agree with you. I didn't think about it from this perspective.